res/qml/Settings/Interface.qml

```diff
@@ -0,0 +1,18 @@
+import QtQuick
+import Mixxx 1.0 as Mixxx
+
+Mixxx.SettingGroup {
+    property list<string> tabs: ["theme & colour", "waveform", "decks"]
```

Comment:
fyi the tabbing stuff here obvsly doesn't work yet, or rather it errors out with 
> Error: Cannot assign to non-existent property "selectedIndex"

Reply:
yeah. I'm also not a huge fan of the approach in the sense that the layouting decisions (such as the virtual tabbing mechanism here) is really controlled by the hosting dialog.
Does that make sense?

Reply:
Yeah, my QML is unfortunately too rusty right now to come up with a good alternative either. I guess it entirely depends on what exactly we want to achieve. I assume search is the primary usecase right now?

Reply:
Yes, that's right. This, as well as the integration with the host popup (e.g pop out of the window and persist the state, or prevent dismissal in change are pending validation)